src/ir_Trotec.cpp

```diff
@@ -0,0 +1,170 @@
+#include "ir_Trotec.h"
+#include <algorithm>
```

Comment:
I don't think you need this header. I don't see anything using it. e.g. max & min etc.
Try it without it.